packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistory.tsx

```diff
@@ -174,11 +190,18 @@ export const TransactionHistory = ({
 
       {/* Transaction history table */}
       <div>
-        <Tab.Group onChange={handleSentOrReceivedTxForSCW} key={address}>
+        <Tab.Group
+          onChange={index => {
+            handleSentOrReceivedTxForSCW(index)
```

Comment:
nit: `handleSentOrReceivedTxForSCW` is not an accurate name anymore. Maybe we should go with something like `handleSmartContractWalletTxHistoryTab` (maybe something shorter but kinda want to be clear what it does)